getting-started/GettingStarted.md

```diff
@@ -0,0 +1,48 @@
+* Must be actively engaged.
+
+### ISV expectations
+If the necessary steps are followed, you can publish the application within 1-2 weeks! We expect you to be actively engaged and contact us if you face any necessary roadblocks after following the required steps. We appreciate your participation in our program. However, if we notice that your engagement levels have decreased, we may need to give priority to other participants due to the high volume of interest we receive.
+
+### Training
+The Following training materials will help accelerate your onboarding onto the Azure Marketplace.
+* [Mastering Container offers for Kubernetes Apps](https://microsoft.github.io/Mastering-the-Marketplace/container/)
+* [Azure K8s Apps Marketplace Demo Walkthrough](https://www.youtube.com/watch?v=_6yGXUND43s)
+
+
+### Steps to follow
+#### Step 1: Package your application
+This step allows you to package your helm chart-based application and other artifacts such as Azure Resource Manager (ARM) template, manifest file, Helm chart and createUIDef into a Cloud Native Application Bundles (CNAB) package. Please refer to the links below:
+* [Prepare your Kubernetes Apps offer](https://learn.microsoft.com/en-us/partner-center/marketplace/azure-container-technical-assets-kubernetes?tabs=windows)
+
+At the end of this step, you will be able to create a CNAB bundle.
+
+#### Step 2: Create the Kubernetes offer
+This step allows you to create an Azure Container offer within Partner Center. You can preview and test the offer before publishing it on the Azure Marketplace.
+* [Create an Azure Container offer](https://learn.microsoft.com/en-us/partner-center/marketplace/azure-container-offer-setup)
+
+In order to simplify the process of Vulnerability scanning and certification, Microsoft strongly recommends you follow the steps in the Vulnerability Scan-Microsoft Defender document to understand the vulnerabilities flagged. Please take action to resolve them before pushing the offer through the pipeline.
+* [Vulnerability Scan-Microsoft Defender](https://github.com/bobmital/kubernetes-offer-samples/blob/main/getting-started/Vulnerability-Scan.md)
+
+#### Step 3: Purchase/Deploy your application
+This step walks through the process that an end consumer would go through to deploy the application onto their Kubernetes cluster.
+* [Deploy a container offer from Azure Marketplace (preview)](https://learn.microsoft.com/en-us/azure/aks/deploy-marketplace)
+
+#### Step 4: If you plan to stop activity on your offer, you can proceed by Stop selling the app. The "Stop Sell" of an offer/plan means no future acquisitions of the offer/plan are possible via Azure Marketplace. Since these are consumption-based plans for now, the existing deployments will continue to emit usage till customers uninstall the extension.
+* [Stop distribution of an offer or plan](https://learn.microsoft.com/en-us/partner-center/marketplace/update-existing-offer#stop-distribution-of-an-offer-or-plan)
+
```

Comment:
Shall we also add troubleshooting links here?
https://learn.microsoft.com/en-us/partner-center/marketplace/azure-container-packaging-troubleshoot
https://learn.microsoft.com/en-us/partner-center/marketplace/azure-container-troubleshoot
https://learn.microsoft.com/en-us/troubleshoot/azure/azure-kubernetes/troubleshoot-failed-kubernetes-deployment-offer

Reply:
Added!